Symfony/CS/ConfigInterface.php

```diff
@@ -128,4 +137,13 @@ public function getCacheFile();
      * @return string|null
      */
     public function getPhpExecutable();
+
+    /**
+     * Set the fixers to be used.
+     *
+     * @param FixerInterface[] $fixers
```

Comment:
>I was thinking we should still pass an array of fixer names in here, and resolve them in the container later on.

I like that suggestion. I'm closing this PR for now because I've to think about how to do this and then find some time to implement it.